modules/administration/pages/auth-methods-sso-example.adoc

```diff
@@ -1,7 +1,8 @@
 [[auth-methods-sso-example]]
 = Example SSO Implementation
 
-In this example, SSO is implemented by exposing three endpoints with {productname}, and using Keycloak as the identity service provider (IdP).
+In this example, SSO is implemented by exposing three endpoints with {productname}, and using Keycloak 9.0.2
+as the identity service provider (IdP).
```

Comment:
Do we actually need the version number here?  If yes, I propose to add "or later".  Additionally, we have a long-line-policy (1 line per sentence); this helps segmenting translation templates.
```suggestion
In this example, SSO is implemented by exposing three endpoints with {productname}, and using Keycloak 9.0.2 or later as the identity service provider (IdP).
```